storage/backup-restore/node-A/nuts.yaml

```diff
@@ -6,17 +6,17 @@ auth:
     - dummy
   irma:
     autoupdateschemas: false
-network:
-  grpcaddr:	:5555
-  certfile: /opt/nuts/certificate-and-key.pem
-  certkeyfile: /opt/nuts/certificate-and-key.pem
-  truststorefile: /opt/nuts/truststore.pem
 tls:
   truststorefile: /opt/nuts/truststore.pem
   certfile: /opt/nuts/certificate-and-key.pem
   certkeyfile: /opt/nuts/certificate-and-key.pem
-
 storage:
   bbolt:
     backup:
-      directory: /backup
\ No newline at end of file
+      directory: /backup
+network:
```

Comment:
moved network the bottom of the config so the `nodedid` can be appended to the file